libraries/app/include/graphene/app/database_api.hpp

```diff
@@ -602,6 +603,54 @@ class database_api
        */
       uint64_t get_son_count()const;
 
+      /////////////////////////
+      // Sidechain Addresses //
+      /////////////////////////
+
+      /**
+       * @brief Get a list of all sidechain addresses
+       * @return The list of all sidechain accounts
+       *
+       * This function has semantics identical to @ref get_objects
```

Comment:
I don't think so, `get_objects` gets a vector as an argument, this does not.

Reply:
Removed